### PR TITLE
Everything converted

### PR DESCRIPTION
**Pull request by thedillonb**

### CODE REVIEW

File: src/components/EnhancedTable/EnhancedTable.tsx [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+      if (orderDirection === 'desc') {
+        return naturalSort(_.toString(b), _.toString(a));
+      } else {
+        return naturalSort(_.toString(a), _.toString(b));
+      }
+    });
+  }, [data, sortParams]);
+
+  const onHeaderClick = React.useCallback((col: IMuiVirtualizedTableColumn<TRow>) => {
```
Comment (benbeshel) [inline src/components/EnhancedTable/EnhancedTable.tsx]: this no longer takes into account disableSort

File: src/components/EnhancedTable/EnhancedTable.tsx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+      if (s && s.orderBy === col.name) {
+        return { ...s, orderDirection: s.orderDirection === 'asc' ? 'desc' : 'asc' };
+      }
+
+      return {
+        orderBy: col.name as keyof TRow,
+        orderDirection: 'desc'
+      };
+    });
+  }, []);
```
Comment (benbeshel) [inline src/components/EnhancedTable/EnhancedTable.tsx]: No dependencies?

File: src/components/EnhancedTable/EnhancedTable.tsx [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+  width: propWidth,
+  height: propHeight,
+  maxHeight,
+  ...tableProps
+}: IProps<TRow>) {
+  type SortParams = { orderBy: keyof TRow; orderDirection?: 'asc' | 'desc' };
+  const [sortParams, setSortParams] = React.useState<SortParams | null>(
+    defaultOrderBy
+      ? {
+          orderBy: defaultOrderBy as keyof TRow,
```
[the remark this answers is not in the document]
Comment (thedillonb) [reply src/components/EnhancedTable/EnhancedTable.tsx]: I'll double check. I think I had tried that and something terrible happened but can't remember what.

File: src/components/EnhancedTable/EnhancedTable.tsx (hunk 2 above)
Comment (thedillonb) [reply src/components/EnhancedTable/EnhancedTable.tsx]: None needed.

File: src/components/EnhancedTable/EnhancedTable.tsx (hunk 1 above)
Comment (thedillonb) [reply src/components/EnhancedTable/EnhancedTable.tsx]: Hmm good point. lemme check

Comment (thedillonb) [reply src/components/EnhancedTable/EnhancedTable.tsx]: Ah right, so `disableSort` was placed on the column but that causes all kinds of type issues. I'll have to figure out another way to restore that functionality.

**Pull request merged**